packages/sdk/src/oracle/services/index.ts

```diff
@@ -1,3 +1,3 @@
 export * as erc20 from "./erc20";
 export * as optimisticOracle from "./optimisticOracle";
-export * as statemachine from "./statemachine";
+export * as statemachines from "./statemachines";
```

Comment:
this was just renamed because there are multiple state machine contexts in this folder. having trouble coming up with good naming conventions